src/tasks/runstart.ts

```diff
@@ -189,6 +192,31 @@ export const RunStartQuest: Quest = {
       do: () => cliExecute("numberology 69"),
       limit: { tries: 3 },
     },
+    {
+      name: "Summon Sugar Sheets",
+      ready: () =>
+        haveSkill($skill`Summon Sugar Sheets`) &&
+        get("_sugarSummons") !== 3 &&
+        (!haveSkill($skill`Summon Clip Art`) || get("instant_saveClipArt", false)),
+      completed: () => have($item`sugar sheet`, 3),
+      do: (): void => {
+        restoreMp(6);
+        useSkill($skill`Summon Sugar Sheets`, 3);
+      },
+      limit: { tries: 1 },
+    },
```

Comment:
This completion condition is also wrong. I don't think we need the `ready()` - the completion condition should probably read
```
!have($skill`Summon Sugar Sheets`) ||
get("_tomeSummons") >= 3 || // use tomeSummons instead of sugarSummons
(have($skill`Summon Clip Art`) && !get("instant_saveClipArt", false))
```
This also affords us more flexibility in tome summons down the road. Furthermore, even if we chose to save Clip Art, we will still use 1 on the borrowed time. This means that you will never have 3 sugar sheets if you do have a Clip Art tome.

the `do()` should thus either summon one sugar sheet at a time (with increased limits/tries), or `3 - get("_tomeSummons")` summons at one go.

We should also add a pref for saving the sugar sheet summons.

Reply:
Done, including adding an `instant_saveSugar` pref